### PR TITLE
Implement pagination for the TwentyTwenty theme

### LINKED ISSUE #292: Add pagination to twentytwenty-theme
**Feature topic**: [Pagination for the TwentyTwenty theme](https://community.frontity.org/t/pagination-for-the-twentytwenty-theme/1229)
**User Story**: Add pagination

[Sprint Card](https://github.com/orgs/frontity/projects/2#card-33013919)

### PR DESCRIPTION
**Pull request by iamuchejude**
#### Description of what you did:
Implement pagination for the `TwentyTwenty Theme`. Fixes #292 

#### My PR is a:

- 🔝 Improvement

### CODE REVIEW

**Comment by michalczaplinski** [comment]
hey @iamuchejude you asked for my review, but it looks like the branch is still a draft and WIP. Let me know when if I should actually review it! 🙂

File: examples/twentytwenty-theme-example/report.20200219.144203.74758.0.001.json [hunk 1]
```diff
@@ -0,0 +1,767 @@
+
```
Comment (luisherranz) [inline examples/twentytwenty-theme-example/report.20200219.144203.74758.0.001.json]: Please delete this file.

Comment (luisherranz) [reply examples/twentytwenty-theme-example/report.20200219.144203.74758.0.001.json]: Where did it come from, by the way? :)

File: packages/twentytwenty-theme/src/components/archive/archive-pagination.js [hunk 2]
```diff
@@ -2,72 +2,174 @@ import React, { useEffect } from "react";
...
+
+    li {
+      display: inline-block;
+      margin: 0;
+
+      &:not(:last-of-type) {
+        margin-right: 2rem;
+      }
+    }
+  }
```
Comment (luisherranz) [inline packages/twentytwenty-theme/src/components/archive/archive-pagination.js]: Could you please migrate this CSS to Emotion by using either the `css` prop or styled components?

**Review by luisherranz — CHANGES_REQUESTED** [review]
Thanks @iamuchejude, great work 👍😁

Everything looks great but, would you mind moving the CSS that is using classes to Emotion?
That's the way we recommend people to use CSS in Frontity so I guess it'd be nice to do it as well in our starter themes.

Also, merge the branch with `dev` because I fixed the problem with the e2e tests there.

File: packages/twentytwenty-theme/src/components/archive/archive-pagination.js (hunk 2 above)
Comment (michalczaplinski) [reply packages/twentytwenty-theme/src/components/archive/archive-pagination.js]: Just to expand on what Luis meant: You should remove the nested CSS classes like `dir` or `directions` and create a styled component / use the[ `css`](https://emotion.sh/docs/css-prop) prop for each element that uses those CSS classes.

**Review by michalczaplinski — COMMENTED** [review]
Looks fine to me, just adding small comment to what Luis already said.

The `paginate` function is nice and clean too 🙂

File: examples/twentytwenty-theme-example/report.20200219.144203.74758.0.001.json (hunk 1 above)
Comment (iamuchejude) [reply examples/twentytwenty-theme-example/report.20200219.144203.74758.0.001.json]: I really can't say where it came from.

**Review by luisherranz — APPROVED** [review]
Thanks @iamuchejude, this is ready for merge.

I've made some extra improvements to the CSS. Please take a look at the commit and if you have any questions let me know.

**Pull request merged**